pkg/cloud/azure/actuators/machine/actuator.go

```diff
@@ -56,6 +56,7 @@ func NewActuator(params ActuatorParams) *Actuator {
 
 // Create creates a machine and is invoked by the machine controller.
 func (a *Actuator) Create(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
+	defer actuators.Recover()
```

Comment:
This really seems like an anti-pattern

Reply:
Thanks, seems like this masked the actual error on failure :( removed it, might add it later